### ISSUE TITLE
Aspect/modstr/temporal annotation for stative verbs and other concepts with (abstract) rolesets

### ISSUE THREAD

**Issue by Marketa-Lopatkova** [ufal/UMR#8]
- NO difference between stative and action verbs ---> aspect/modstr/temporal annotation (preliminary)!!
- nine abstract predicates (as definedin the Guidelines) ... we use them only if we are sure that it is a predication --> aspect/modstr/temporal annotation (preliminary)!!
- other concepts with abstract rolesets (reification, implicit rolesets) ... the same as for pother -91 "predicates" --> aspect/modstr/temporal annotation (preliminary)!!
- have-quant-91, have-degree-91) ... we use them only if we are sure that it is a predication --> aspect/modstr/temporal annotation (preliminary)!!
- BUT: discourse relations ... NOT events (despite having rolesets)

- agentive nouns are annotated as verbs ---> events, i.e., with aspect/modstr/temporal annotation (preliminary)!! 
        Šárka: a teacher teaches habitually vs. for a murderer, it is enough to commit just one murder to be a murderer.
               Do we want to mark the aspect at the verb to_teach, to_kill? YES!
       
- words denoting processes (válka, příchod, bouře) ---> events, i.e., use valency frame from PDT-Vallex or CREATE it (??where to store them?; NOT for automatic conversion) and annotate aspect/modstr/temporal annotation (preliminary)!!

**Comment by dan-zeman** [comment]
> * BUT: discourse relations ... NOT events (despite heving rolesets)

The validator no longer treats discourse concepts as events (https://github.com/ufal/UMR/commit/1cb51b6f4336e4d3e082c133e411d7b12f545259).

**Comment by dan-zeman** [comment]
@Marketa-Lopatkova you do not mention `publication-91` as receiving special treatment but I think you said it would be exempt from the `:aspect` and `:modal-strength` requirement, too? Though in your excerpt from Julia's answers it sounds still undecided, or maybe leaning towards treating it as an event:

_I think it's much more unusual for a lot of the other implicit rolesets to come up in a predicated form, but some of them do occasionally, and that goes for the -91 rolesets used for reification as well. (Generally, this is a copular or existential construction in English). I can't think of a case with publication-91 offhand-- maybe that one isn't one that occurs in English. My goal is to try to follow the same predicating/non-predicating guidelines with all of the -91 rolesets._

**Comment by Marketa-Lopatkova** [comment]
@dan-zeman you are right: `publication-91` and other abstract rolesets for metadata information should be exempt from the  `:aspect` and `:modal-strength` requirement as well. I've updated the instructions in the [Events file](https://github.com/ufal/UMR/tree/main/doc/eventive-concepts.md).

**Comment by dan-zeman** [comment]
> OK. As of [1b14b96#diff-5f3c94b4ce37d3d41d4d505c19f20bd288ed17c8ae575a988d802f8d6a8c5e10](https://github.com/ufal/UMR/commit/1b14b968f67cdfc3ea998c36dd4063e84e8f763d#diff-5f3c94b4ce37d3d41d4d505c19f20bd288ed17c8ae575a988d802f8d6a8c5e10), `publication-91` is not considered event and does not require `:aspect` and `:modal-strength`.
> 
> Should we go to the other extreme and disallow these roles with this concept?

It is banned [in the validator](https://github.com/ufal/UMR/blob/18bbfa7d456d7bdc24f698bc156caffaef371128/tools/validate.py#L1520-L1526). So I'm closing the issu.
